src/main/scala/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -21,12 +21,20 @@ trait ClassLikeSupport:
 
     val (extensions, methods) = classDef.symbol.classMethods.filterNot(isSyntheticFunc).partition(isExtensionMethod(_))
 
-    val parsedExtensions = extensions
-      .map(parseMethod(_, isExtension = true))
+    
+    val parsedExtensions = extensions.map(e => (getExtendedSymbol(e).get, e) ).groupBy{
+      case (arg, e) => arg.symbol.pos
+    }.map{
+      case (pos, list) => if(list.size == 1) {
+        ExtensionGroup(parseArgument(list(0)._1, _ => "", true, false), List(parseMethod(list(0)._2, extInfo = Some(ExtensionInformation(false)))))
```

Comment:
A note about `_1` and similar: in Scala 3 the preferred way to index a tuple is `(1)`, this is as efficient as `_1`.

Reply:
It's a bit tricky as "_1" is actually "(0)" ;)